systemvm/debian/opt/cloud/bin/checkrouter.sh

```diff
@@ -18,13 +18,29 @@
 
 STATUS=UNKNOWN
 
+ROUTER_TYPE=$(cat /etc/cloudstack/cmdline.json | grep type | awk '{print $2;}' | sed -e 's/[,\"]//g')
+if [ "$ROUTER_TYPE" = "vpcrouter" ];then
+    GUEST_NICS=$(python3 -c "
+import json
+data = json.load(open('/etc/cloudstack/ips.json'))
+for nic, objs in data.items():
+   if isinstance(objs, list):
+       for obj in objs:
+           if obj.get('nw_type') == 'guest' and obj.get('add'):
+               print(nic)
+                     ")
```

Comment:
create a method for readability?

Reply:
done. thanks @DaanHoogland for the review